collector/__init__.py

```diff
@@ -364,6 +367,21 @@ def _process_timing_windows(prog: Program, obs: Observation) -> List[Time]:
                 windows.extend([Time([begin + i * period, begin + i * period + duration]) for i in range(repeat)])
 
         return windows
+    
+    @staticmethod
+    def _calculate_proper_motion(target: Target, time: Time) -> SkyCoord:
+        """
+        Calculate the proper motion of a target.
+        """
+
+        pm_ra = target.pm_ra / Collector._MILLIARCSECS_PER_DEGREE
+        pm_dec = target.pm_dec / Collector._MILLIARCSECS_PER_DEGREE
+        
+        time_offsets = time - Collector._EPOCH2TIME[target.epoch]
```

Comment:
Instead of this, we should probably have something like:
```
epoch_time = Collector._EPOCH2TIME.getdefault(target.epoch, Time(target.epoch, format='jyear'))
time_offsets = time - epoch_time
```

This unfortunately requires us to be reliant on AstroPy, but I suspect we can make our own function using `datetime` to do the calculation. It looks like, from https://en.wikipedia.org/wiki/Epoch_(astronomy) that:
```
J = 2000 + (Julian date − 2451545.0) ÷ 365.25
```
where J is the epoch, so we can get the Julian date:
```
jd = (epoch - 2000)  * 365.25 + 2451545.0
```

There is some simple code to convert between Julian dates and `datetime` here, I think:
https://github.com/dannyzed/julian/blob/master/julian/julian.py

that would allow us to avoid AstroPy.